### PR TITLE
[MODEL-12041] Template to show loading a pretrained estimator

### PR DESCRIPTION
**Pull request by scottp-ml**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
This is a simple template that shows how to load a pretrained artifact as part of a custom task.   The template is very simple for clarity in how to use the load method, versus the more complex example with ViT.

### CODE REVIEW

File: task_templates/2_estimators/12_python_pretrained/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Using a pretrained model in Python
+
+This template shows how to utilize a pretrained model in a custom task.  The pretrained model
+in this case is a simple linear regression model trained on the [Juniors 3 year stats dataset](tests/testdata/juniors_3_year_stats_regression.csv).  It
```
Comment (akshoop) [inline task_templates/2_estimators/12_python_pretrained/README.md:4]: This dataset is only available in DR repo and so any non-DR user of DRUM won't have access to this CSV link

File: task_templates/2_estimators/12_python_pretrained/custom.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+"""
+Copyright 2021 DataRobot, Inc. and its affiliates.
```
Comment (akshoop) [inline task_templates/2_estimators/12_python_pretrained/custom.py:2]: nit Copyright year:
```suggestion
Copyright 2023 DataRobot, Inc. and its affiliates.
```

**Review by akshoop — APPROVED** [review]
LGTM from Core Modeling with minor comments, CC @danna-naser @eric-s-s for awareness and more eyes for feedback

File: task_templates/2_estimators/12_python_pretrained/README.md (hunk 1 above)
Comment (scottp-ml) [reply task_templates/2_estimators/12_python_pretrained/README.md:4]: That is part of this (drum) repo, which is not restricted.

Comment (akshoop) [reply task_templates/2_estimators/12_python_pretrained/README.md:4]: Oh gotcha thanks for the clarification

**Review by yakov-g — APPROVED** [review]

**Pull request merged**